io.github.lethal_guitar.RigelEngine.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: io.github.lethal_guitar.RigelEngine
+
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "21.08"
+
+command: RigelEngine
+
+rename-appdata-file: rigelengine.metainfo.xml
+rename-desktop-file: rigelengine.desktop
+rename-icon: rigelengine
+
+finish-args:
+  - --device=all # to access game controllers
+  - --filesystem=home:ro # to let the user place game data in a desired location
```

Comment:
it would be better to restrict to a location.

Reply:
I'll look into it